source/common/request_info/dynamic_metadata_impl.cc

```diff
@@ -6,7 +6,8 @@ namespace Envoy {
 namespace RequestInfo {
 
 void DynamicMetadataImpl::setData(absl::string_view data_name, std::unique_ptr<Object>&& data) {
-  if (data_storage_.find(data_name) != data_storage_.end()) {
+  // TODO(Google): Remove string conversion when fixed internally.
+  if (data_storage_.find(std::string(data_name)) != data_storage_.end()) {
```

Comment:
this function does same conversion 5 lines below, have a variable to store it?

Reply:
Done.